pincer/middleware/interaction_create.py

```diff
@@ -102,16 +102,18 @@ async def interaction_handler(
 
     args = []
 
-    if interaction.data.type is AppCommandType.USER:
+    if interaction.data.type == AppCommandType.USER:
```

Comment:
Why?

Reply:
`Is` doesn't work properly for this. It will consider 1 and 1 from the enum to be different but it's should just be an equality check.